config.example.cfg

```diff
@@ -1,4 +0,0 @@
-DEBUG = True
```

Comment:
Please exclude this file from your commits. It should still be the original file. If you deleted it locally, please create it again using the content from the official repository.